fdbserver/BlobManifest.actor.cpp

```diff
@@ -191,18 +191,19 @@ class BlobManifest {
 	}
 
 	// Delete all files of oldest manifest
-	static void deleteOldest(const std::vector<BlobManifestFile>& allFiles,
-	                         Reference<BackupContainerFileSystem> container) {
+	ACTOR static Future<Void> deleteOldest(std::vector<BlobManifestFile> allFiles,
+	                                       Reference<BackupContainerFileSystem> container) {
 		if (allFiles.empty()) {
-			return;
+			return Void();
 		}
-		int64_t epoch = allFiles.back().epoch;
-		int64_t seqNo = allFiles.back().seqNo;
+		state int64_t epoch = allFiles.back().epoch;
+		state int64_t seqNo = allFiles.back().seqNo;
 		for (auto& f : allFiles) {
 			if (f.epoch == epoch && f.seqNo == seqNo) {
-				container->deleteFile(f.fileName);
+				wait(container->deleteFile(f.fileName));
```

Comment:
nit: would it be useful to define `std::vector<Future<Void>>` and then outside the loop do `waitForAll(futures)`; it might be more performant

Reply:
thanks.. that's a good point. although we only have a few files to be deleted every time.. it still sounds better to use a vector when it scales